rpm-software-management/rpmlint
# Add tests in test_tags & description in TagsCheck.toml

This commit contains,
- Add test in test_tags.py
    - test_check_requires_on_release
    - test_check_invalid_license
    - test_package_not_std_release_extension
    - test_check_non_standard_group
    - test_package_dev_dependency
- Add description of non-standard-group check in
TagsCheck.toml
- Add packages in test/binary/
